src/nandini/FibonacciNumber.java

```diff
@@ -0,0 +1,26 @@
+package nandini;
+
+public class FibonacciNumber {
+
+	void numberIsFibonacci(int num) {
+		int a = 0;
+        int b = 1;
+        if (num==a || num==b) ;
```

Comment:
what is num has value 0 or 1. you putted sime-column ";" after the if condition.